api/nodes/views.py

```diff
@@ -232,12 +236,38 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
-    # overrides ListCreateAPIView
+    # overrides ListBulkCreateJSONAPIView, BulkUpdateJSONAPIView
     def get_queryset(self):
-        query = self.get_query_from_request()
-        return Node.find(query)
+        # For bulk requests, queryset is formed from request body.
+        if is_bulk_request(self.request):
```

Comment:
People *really* don't like JSON with get. If we really want to have a bulk get capability, which we might, we could special case filtering on id. `?filter[id]=[ngje1,or443,023kk]` or similar.